bitnami/thanos/templates/ruler/service.yaml

```
@@ -5,11 +5,15 @@ metadata:
   name: {{ include "thanos.fullname" . }}-ruler
   labels: {{- include "thanos.labels" . | nindent 4 }}
     app.kubernetes.io/component: ruler
+    monitor: "true"
```

When using Prometheus (instead of Prometheus Operator), you can manage what endpoints to scrape using the annotation: `prometheus.io/scrape: 'true'`. However, annotations are ignored when using the Prometheus Operator.

We can use a similar naming here for this label: `prometheus-operator/monitor: 'true'`

```suggestion
    {{- if .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
    prometheus-operator/monitor: 'true'
    {{- end }}
```